src/core/sys/freebsd/sys/event.d

```diff
@@ -15,6 +15,8 @@ module core.sys.freebsd.sys.event;
 
 version (FreeBSD):
 extern (C):
+nothrow:
+@nogc:
```

Comment:
Be careful with those in C headers, sometime callback types are declared which would become more restrictive by that and a deprecated overload taking the old callbacks would be in order.